### PR TITLE
CDAP-14515 Update doc to drop java 7 from requirements

### PR DESCRIPTION
**Pull request by rohitsinha54**
Should be ported back to 5.0 docs too. 

Note: Merge pending on version bump to 5.1.1-SNAPSHOT if needed.

### CODE REVIEW

**Comment by dereklwood** [comment]
lgtm, but there is a reference [here](https://docs.cdap.io/cdap/5.1.0/en/developer-manual/getting-started/sandbox/index.html) as well

**Comment by dereklwood** [comment]
and [here](https://docs.cdap.io/cdap/5.1.0/en/faqs/general.html#what-version-of-java-sdk-is-required-by-cdap)

**Comment by rohitsinha54** [comment]
@dereklwood : Addressed comments. Please have another look.

**Comment by rohitsinha54** [comment]
Build: https://builds.cask.co/browse/CDAP-DQB439-1

File: cdap-docs/admin-manual/source/system-requirements.rst [hunk 1]
```diff
@@ -99,7 +99,7 @@ You'll need this software installed:
 
 Java Runtime
 ------------
-The latest `JDK or JRE version 1.7.xx or 1.8.xx <http://www.java.com/en/download/manual.jsp>`__
+The latest `JDK or JRE version 1.8.xx <http://www.java.com/en/download/manual.jsp>`__
```
Comment (anwar6953) [inline cdap-docs/admin-manual/source/system-requirements.rst:102]: Aren't there after 1.8.x that we support? Like, Java 10.

Comment (anwar6953) [reply cdap-docs/admin-manual/source/system-requirements.rst:102]: Ah nevermind; we don't currently support Java 10:
https://issues.cask.co/browse/CDAP-14312

**Review by dereklwood — APPROVED** [review]

**Comment by rohitsinha54** [comment]
Update the jira is : https://issues.cask.co/browse/CDAP-14514

**Pull request merged**